python/ucxx/_lib_async/tests/test_send_recv_am.py

```diff
@@ -83,13 +83,17 @@ async def test_send_recv_am(size, recv_wait, data):
             # immediately as receive data is already available.
             await asyncio.sleep(1)
         await c.am_send(msg)
-    for c in clients:
-        await c.close()
-    await wait_listener_client_handlers(listener)
+
+    while len(recv) == 0:
+        await asyncio.sleep(0)
 
     if data["memory_type"] == "cuda" and msg.nbytes < rndv_thresh:
         # Eager messages are always received on the host, if no custom host
         # allocator is registered, UCXX defaults to `np.array`.
         np.testing.assert_equal(recv[0].view(np.int64), msg.get())
     else:
         data["validator"](recv[0], msg)
+
+    for c in clients:
+        await c.close()
```

Comment:
Is it better to `await asyncio.gather(*(c.close() for c in clients))` ?

Reply:
Should be equivalent, but I agree your suggestion is a more asyncio-nic solution, done in https://github.com/rapidsai/ucxx/pull/58/commits/4f3d5d819fefdbd25d857ac5c02547b55d41c750.